.gitignore

```
@@ -1,2 +1,4 @@
 /nbproject/
-/target/
\ No newline at end of file
+/target/
+.idea/
+jchess.iml
```

I don't see why `jchess.iml` should be ignored :thinking:

```suggestion
/.idea/
```

---

I usually do that but I don't even need it here. Removed.